.apirc

```diff
@@ -0,0 +1,16 @@
+{
+  "persistence": {
+    "postgres": {
+      "connection": {
+        "host": "localhost",
+        "user": "postgres",
+        "password": "postgres",
+        "database": "test_db"
+      },
+      "pool": {
+        "min": "2",
+        "max": "7"
+      }
+    }
+  }
+}
```

Comment:
What's this file for? From the code it seems like you use it to set the connection for knex, but if this is configuration that we may need to change for new projects, it would be good to mention in the readme.

Reply:
added a line in README explaining this